app/js/components/cohorts/actionsComponent.jsx

```diff
@@ -125,11 +130,21 @@ class ActionsComponent extends Component {
 		};
 	}
 
+    deleteCohort(cohortId) {
+        return (e) => {
+            const apiHelper = new ApiHelper();
+            apiHelper.delete(`/cohort/${cohortId}`)
+                .then(res => {
+                    this.getAllCohorts();
```

Comment:
True; for now, we're not using it but I feel there may be a need to use it in the future. If we want to display a form of confirmation later on then we may need to check some properties of `res` to determine the right information to display